test/e2e/frontend/cypress/support/commands.js

```diff
@@ -1,3 +1,5 @@
+import 'cypress-mailpit'
```

Comment:
Does this actually do anything?

Reply:
not yet, it just adds convenience methods for cypress to interact with the smtp api. The commands will be used when creating tests in #4020

Reply:
OK, but I'm surprised this passes the lint test if it's not being used yet.

Reply:
It must be because I'm not storing anything in unused const/let just extending